package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "prettier": "1.16.4",
+    "prettier-check": "2.0.0"
+  },
+  "scripts": {
+    "prettier-check": "prettier-check **/*"
```

Comment:
Why `prettier-check` instead of simply `prettier --check`? One dependency would be much more preferably to two, if possible. Two opens up the possibility for conflicts and means Renovate has more to do.